superset-frontend/plugins/plugin-chart-table/src/utils/extent.ts

```diff
@@ -16,7 +16,6 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
```

Comment:
Nit: I assume this is a mistake - let's avoid touching unrelated files.

Reply:
Good catch. Removed changes